modules/e2ee/Worker.js

```diff
@@ -7,18 +7,29 @@ import { Context } from './Context';
 
 const contexts = new Map(); // Map participant id => context
 
+let singleKey = false;
+let singleContext = undefined;
+
 /**
  * Retrieves the participant {@code Context}, creating it if necessary.
  *
  * @param {string} participantId - The participant whose context we need.
  * @returns {Object} The context.
  */
 function getParticipantContext(participantId) {
-    if (!contexts.has(participantId)) {
-        contexts.set(participantId, new Context(participantId));
-    }
+   if (singleKey) {
+       if (!singleContext) {
+          singleContext = new Context();
```

Comment:
Instead of doing it here, implicitly, let's be explicit and have a `initialize` method, which sets it  up.